<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIAppClass</class>
 <widget class="QMainWindow" name="GUIAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Szukaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="analyzeButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Analiza</string>
    </property>
   </widget>
   <widget class="QListWidget" name="resultList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>135</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wprowadź nazwę folderu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="folderNameEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>126</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wprowadź rozszerzenie</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="extensionLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>108</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sciezkadostepu">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUIApp.qrc"/>
 </resources>
 <connections/>
</ui>
